parts/linux/cloud-init/artifacts/cse_config.sh

```diff
@@ -449,7 +449,7 @@ configureKubeletServing() {
 
     # check if kubelet serving certificate rotation is disabled by customer-specified nodepool tags
     export -f should_disable_kubelet_serving_certificate_rotation
-    DISABLE_KUBELET_SERVING_CERTIFICATE_ROTATION=$(retrycmd_if_failure_no_stats 10 1 10 bash -cx should_disable_kubelet_serving_certificate_rotation)
+    DISABLE_KUBELET_SERVING_CERTIFICATE_ROTATION=$(retrycmd_silent 10 1 10 bash -cx should_disable_kubelet_serving_certificate_rotation)
```

Comment:
pls make sure these e2e scenarios pass

Reply:
Yep will be verifying everything once i run generate + fix conflicts

Reply:
Looks like they are passing, only failure was for A100 GPU test
![image](https://github.com/user-attachments/assets/61a63068-b39b-415e-ab37-05b85bbcf83b)